src/mds/InoTable.cc

```diff
@@ -111,6 +113,8 @@ void InoTable::replay_alloc_id(inodeno_t id)
 }
 void InoTable::replay_alloc_ids(interval_set<inodeno_t>& ids) 
 {
+  assert(mds);  // Only usable in online mode
+
```

Comment:
This is because previously the class was only ever used with a non-null `mds` attribute, whereas now it gets used with a null one sometimes, so the assertions were to make that less unsafe.  However... I cannot think why I only added assertions to these two functions.